src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum;
+  const dayilyPay = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < 3) {
+    sum = days * dayilyPay;
+  };
+
+  if (days > 2 && days < 7) {
+    sum = days * dayilyPay - discountSmall;
+  };
+
+  if (days > 6) {
+    sum = days * dayilyPay - discountBig;
+  }
+
+  return sum;
```

Comment:
```suggestion
    return (days * dayilyPay - discountBig);
```

no need for that last check, because if the first two checks won't pass, so the function will return this result by default